libs/pika/execution/include/pika/execution/algorithms/drop_operation_state.hpp

```diff
@@ -248,5 +249,14 @@ namespace pika::execution::experimental {
         {
             return detail::partial_algorithm<drop_operation_state_t>{};
         }
-    } drop_operation_state{};
+    };
+
+    /// \brief Release any operation state coming before the adaptor before it signals a connected
+    ///  receiver.
```

Comment:
Possibly shorter:
```suggestion
    /// \brief Releases the operation state of the adaptor before signaling a connected receiver
```
?